lib/folio_client.rb

```diff
@@ -123,4 +126,9 @@ def data_import(...)
       .new(self)
       .import(...)
   end
+
+  def holdings(...)
+    Holdings
```

Comment:
It looks like this is the source of the coverage drop.

Reply:
Thanks. Adding tests.